Fixing my mess

benjaminlong opened the pull request:
En résumé:

- WIP: Added simple README & Added comment using copilot
J'ai juste ajouté des commentaires pour améliorer la lisibilité et ajouté un README pour décrire le projet et avoir à dispo les commandes pour tester et lancer le projet par le terminal.

- WIP: Renamed Hand model to Player with pk, name and an hand.
J'ai juste renommé le model Hand en Player.

- 3eme:
Simplifié l'écriture de la distribution des cartes. 
Mise au propre de la fin du fichier battle.py. J'ai remis en place ton lancement du jeu, utilisant ton algo de comparaison.
Le projet remarche ^^

- 4-5-6eme: 
C'est juste de la mise au propre (réparation des test-unitaires, mise a jour de la doc, mise en place des modules pour pouvoir séparé le fichier battle.py en plusieurs fichiers. Ex: chargement du fichier models.py).

 Dans l'onglet "Commits", tu peux ensuite voir pour chaque commit les différents changements.
 A gauche l'ancien fichier, A droite le nouveau. C'est toujours un très bon exercice d'arriver a lire chaque commit, mais pas toujours si simple...

benjaminlong committed:
WIP: Added simple README & Added comment using copilot

benjaminlong committed:
WIP: Renamed Hand model to Player with pk, name and an hand.

benjaminlong committed:
WIP: Removed and Fixed my code to match cecilia while loop implementation

- The play function will be refactorized later.

benjaminlong committed:
ENH: Added a full .gitignore file

benjaminlong committed:
TEST: Fixed pytest

benjaminlong committed:
DOC: Updated README file with a working Usage CLI "battle.py"

benjaminlong committed:
DOC: Updated README file with a working Usage CLI "battle.py"

- Moving Card model to models.py file to test module mode.
- Updated README file with new battle CLI command.

ceciliabl reviewed (approved):
j'ai tout compris sur celui la  ;)

benjaminlong commented:
Reste plus qu'a faire la version The Crew ^^

benjaminlong committed:
WIP: Few updates to factorize prior to start the crew game

- Move all models to models.py file.
- Simplify Deck model and created decks.py file to separate creation
logic from Deck model. There are too many ways to create a deck...
- Added diplay_name attribute to Card model.
- Added utils.py file.
- Added simple UTs.
- Few imports fixes & Cleaned unused file "corsica_battle".

benjaminlong committed:
WIP: adding display_current_state method to various models to ease logs

benjaminlong committed:
TBC: To Be Continued...
